aspnetcore/signalr/get-started-signalr-core.md

```diff
@@ -46,30 +58,60 @@ Install the following software:
 
   ![New Project dialog in Visual Studio](get-started-signalr-core/_static/signalr-new-project-choose-type.png)
 
-  The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
+The libraries that host SignalR server-side code are included in the project template. Install the client-side JavaScript separately with [npm](https://www.npmjs.com/).
 
   ```console
    npm install @aspnet/signalr
   ```
 
 3. Copy the *signalr.js* from *node_modules\\@aspnet\signalr\dist\browser* to the *wwwroot\lib* folder in your project.
 
+# [Visual Studio Code](#tab/visual-studio-code)
+
+1. From a terminal, run the following commands:
+ 
+  ```console
+  mkdir SignalRChat
+  cd SignalRChat
+  dotnet new SignalRChat razor
+  ```
+
+-----
+
 ## Create the SignalR Hub
 
 A hub is a class that serves as a high-level pipeline that allows the client and server to call methods on each other.
 
-1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**. 
+# [Visual Studio](#tab/visual-studio)
+
+1. Add a class to the project by choosing **File** > **New** > **File** and selecting **Visual C# Class**.
 
 1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data.
 
-1. Create the `Send` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
+1. Create the `SendMessage` method that sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
+
+  [!code-csharp[Startup](get-started-signalr-core/sample/Hubs/ChatHub.cs?range=7-14)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+1. Open the *SignalRChat* folder in Visual Studio Code (VS Code).
+
+1. Add a class to the project by selecting **File** > **New File** from the menu.
+
+1. Inherit from `Microsoft.AspNetCore.SignalR.Hub`. The `Hub` class contains properties and events for managing connections and groups, as well as sending and receiving data to clients.
+
+1. Add a `SendMessage` method to the class. The `SendMessage` method sends a message to all connected chat clients. Notice it returns a `Task`, because SignalR is asynchronous. Asynchronous code scales better.
 
   [!code-csharp[Startup](get-started-signalr-core/sample/Hubs/ChatHub.cs?range=7-14)]
 
+-----
+
 ## Configure the project to use SignalR
 
 The SignalR server must be configured so that it knows to pass requests to SignalR.
 
+# [Visual Studio](#tab/visual-studio)
+
 1. To configure a SignalR project, modify the `ConfigureServices` method of the application's `Startup` class by inserting a call to `services.AddSignalR`.
```

Comment:
~~modify the `ConfigureServices` method of the application's `Startup` class~~
modify the project's `Startup.ConfigureServices` method